### ISSUE TITLE
Names not appearing

### ISSUE THREAD

**Issue by shaananc** [monicahq/monica#1497]
**Describe the bug**
Self hosting using provided docker image.
In the list of contacts, or in an individual contact view the names themselves don't appear

Unsure how to check logs.

**Screenshots**
<img width="982" alt="screen shot 2018-06-19 at 8 14 33 am" src="https://user-images.githubusercontent.com/2287839/41596621-ed4c92a6-7398-11e8-80a1-1a675f6a4dc3.png">

**Comment by djaiss** [comment]
Yep this is a nasty bug that will be fixed this week. The pull request is already made.

**Comment by shaananc** [comment]
What pull request number is it?

**Comment by djaiss** [comment]
Exactly here https://github.com/monicahq/monica/pull/1491/files#diff-fdfcde0ece1a5848c705846866e898fbR16

**Comment by shaananc** [comment]
Doesn't seem to have fixed it on my install.

**Comment by djaiss** [comment]
Can you check in your database in the Users table, and check the content of `name_order` for your user, and report it here?

**Comment by djaiss** [comment]
For sure the bug is linked to this field having an invalid value.

**Comment by shaananc** [comment]
```
+-----------------+
| name_order      |
+-----------------+
| firstname_first |
+-----------------+

**Comment by shaananc** [comment]
Manually updating the value to `firstname_lastname_nickname` worked but the migration didn't automatically fix my existing user.

I'm not planning on digging into the codebase to fix it myself but hopefully this helps!

**Comment by oschettler** [comment]
Setting (German) Namensortierrichtung (name sort order) on /settings to First-Last (Nick) fixed this for me

**Comment by qbatten** [comment]
This bug was appearing for my install as well. As @oschettler said, going to the Settings page and changing the Name Order setting to First Name - Last Name - Nickname fixes it!

**Comment by asbiin** [comment]
@djaiss shouldn't it be easier to fix this outside the #1491 PR ?

**Comment by tmuras** [comment]
Note: I've hit the same problem after installing v2.3.1. Upgrade to v2.4.2 did not help.
